### DISCUSSION TITLE
Incorrect Placement of Commas in Total Supply Display for CATs

### DISCUSSION THREAD

**Discussion by P-r-a-s-a-n-n-a** [spacescan-io/Spacescan-discussion#75] [Support]
![Screenshot_20230808-212419_Discord](https://github.com/spacescan-io/Spacescan-discussion/assets/89355713/a3d69581-2af8-46f1-a84f-9864261aff08)

**Comment by CheebaWizard** [comment]
I was about to mention on the discord, but probably unnecessary. This is really Max Supply as well. Total Supply would exclude any burnt coins. 😉

**Reply by CheebaWizard** [reply]
Would be a cool metric to include just need reduce by the amounts that turn up in:

https://spacescan.io/address/xch1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqm6ks6e8mvy
https://spacescan.io/address/xch1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq2u30kz

**Reply by CheebaWizard** [reply]
Could then also work out a fully diluted market cap 😜

**Reply by natsabari** [reply]
@CheebaWizard  Sure, that's good one and will add it to our pipeline.
